core/App/screens/CredentialOffer.tsx

```diff
@@ -159,6 +160,7 @@ const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route })
             }}
           >
             {loading ? <RecordLoading /> : null}
+            <ConnectionNotification connectionID={credentialConnectionLabel} />
```

Comment:
Do we want to check for the truthiness of the `credentialConnectionLabel` like what's done with the `proofConnectionLabel`?

Reply:
sometimes on the BC Wallet Demo the connection is formed but there's an error issuing the credential and the user has to restart the app and press the `resend credential` button on the demo. In this case the connectionID is sometimes blank even though a connection is formed. I could check for truthiness but I figured since a connection is always formed when getting a credential, the notification should always be on the credential offer page